examples/platform_channel/macos/RunnerTests/RunnerTests.swift

```diff
@@ -0,0 +1,16 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import FlutterMacOS
+import Cocoa
+import XCTest
+
+class RunnerTests: XCTestCase {
```

Comment:
I think either delete this or test it.

Reply:
Left it in for consistency with the iOS sample, but I'll send a patch to delete that as well in a sec.

Reply:
Ah maybe not -- looks like an old file sitting around. Cleaned out with git clean and it's gone.